specification.md

```diff
@@ -3258,7 +3326,7 @@ If events defined in event-based conditions do not arrive before the states `eve
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
 | name | Data condition name | string | no |
-| [condition](#Workflow-Expressions) | Workflow expression evaluated against state data. Must evaluate to true or false | string | yes |
+| [condition](#Workflow-Expressions) | Boolean or workflow expression that evaluates to boolean against state data | boolean or expression | yes |
```

Comment:
again there is no "boolean or workflow expression" its just an expression that must return boolean

Reply:
again, can be set to json true/false if eventually we only care that what is in here is boolean.